frontend/src/features/logic/utils/isConditionFulfilled.ts

```diff
@@ -41,9 +41,19 @@ export const isConditionFulfilled = (
 
   switch (condition.state) {
     case LogicConditionState.Lte:
-      return Number(currentValue) <= Number(condition.value)
+      if (!currentValue)
```

Comment:
what if `currentValue` is 0? then this check will always be true, which is not what we want right

Reply:
also should we insert this check on line 40 instead? we're already checking for `undefined`, so perhaps we should be checking for `null` as well? can reference `isConditionFulfilled` in `src/shared/util/logic.ts` for the current battle-tested implementation - not sure why we're not reusing that tbh (@karrui any idea?)

Reply:
> what if currentValue is 0? then this check will always be true, which is not what we want right
yes good catch will fix!

Reply:
> also should we insert this check on line 40 instead? we're already checking for `undefined`, so perhaps we should be checking for `null` as well? can reference `isConditionFulfilled` in `src/shared/util/logic.ts` for the current battle-tested implementation - not sure why we're not reusing that tbh (@karrui any idea?)

src/shared is using backend types, so I had to copy and paste the code. At least I thought I copied them properly...